app/controllers/user_submissions_controller.rb

```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+
+class UserSubmissionsController < ApplicationController
+  before_action :user_required  
+
+  def index
+    @submissions = current_person.authorships.map(&:submission)
+  end
+
+  def show
```

Comment:
Tutaj niestety źle Pani wskazałem poprawki - dopiero jak sobie to uruchomiłem to uświadomiłem sobie, że to nie jest do końca poprawne. Przetestowany i poprawny kod wygląda następująco:
```
authorship = current_person.authorships.where(submission_id: params[:id]).first
if authorship.nil?
  redirect_to user_submissions_path
  return
end
@submission = authorship.submission
```